enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/QueryStatusResult.java

```diff
@@ -43,10 +43,22 @@
     public final Map<String,Object> parameters;
     public final String startTime;
     public final String elapsedTime;
+    /** EXPERIMENTAL: added in Neo4j 3.2 */
```

Comment:
We will be documenting the experimental fields separately as part of procedure docs. We wanted to release this in a milestone to get some feedback and have the ability to modify based on feedback. We will remove this once we finish with the implementation/ in the next major release.